hackathon_site/event/models.py

```diff
@@ -18,6 +18,8 @@ class Team(models.Model):
     created_at = models.DateTimeField(auto_now_add=True, null=False)
     updated_at = models.DateTimeField(auto_now=True, null=False)
 
+    project_description = models.CharField(max_length=500, null=True)
```

Comment:
Should this be a nullable field? During hackathons we might want to enforce hackers to give us a mandatory short description of their project, cause that would help admins give them what they need

Reply:
Since you are changing the model, and the change is minor, delete the [0008_team_project_description.py](https://github.com/ieeeuoft/hackathon-template/pull/499/files#diff-a12e3b9d7090057f3fe961c96da6a3e559a116c4faf238c40edcbf6e26eef342), and create a new migration so we only have one migration for the project_description field addition. Let me know if you need further clarification.

Reply:
@KarandeepLubana Nevermind please keep the project description as null=True because we don't want teams to have to write a project description when they first create a team. We can enforce users to write a project description on the frontend with React.